demonstrations/tutorial_classical_kernels.py

```diff
@@ -0,0 +1,738 @@
+r"""
+
+.. _classical_kernels:
+
+How to approximate a classical kernel with a quantum computer
+=======================================================================
+
+.. meta::
+    :property="og:description": Finding a QK to approximate the Gaussian
+    kernel.
+    :property="og:image": https://pennylane.ai/qml/_images/toy_qek.png
+
+.. related::
+
+   tutorial_kernels_module Training and evaluating quantum kernels
+   tutorial_kernel_based_training Kernel-based training of quantum models with
+   scikit-learn
+   tutorial_expressivity_fourier_series Quantum models as Fourier series
+
+
+*Author: Elies Gil-Fuster (Xanadu resident). Posted: DD MMMM 2021.*
+
+Forget about advantages, supremacies, or speed-ups.
+Let us understand better what we can and cannot do with a quantum computer.
+More specifically, in this demo we want to look into quantum kernels and ask
+whether we can replicate classical kernel functions with a quantum computer.
+Lots of researchers have lengthily stared at the opposite question, namely that
+of classical simulation of quantum algorithms.
+Yet, by studying what classes of functions we can realize with quantum kernels,
+we can gain some insight into their inner workings.
+
+Usually, in quantum machine learning (QML) we use parametrized quantum circuits
+(PQCs) to find good functions, whatever *good* means here.
+Consequently, since kernels are just one specific kind of well-defined
+functions, the task of finding a quantum kernel (QK) that approximates a given
+classical one could be posed as a supervised learning regression problem.
```

Comment:
```suggestion
classical one could be posed as an optimisation problem.
```

Would you mind this change? Learning is one option to match kernels via optimisation, but this is more general...

Reply:
yes makes sense